app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/viewmodel/ParkingStayViewModel.kt

```diff
@@ -59,6 +60,15 @@ class ParkingStayViewModel(private val model: ParkingStayModel) {
         }
     }
 
+    fun saveReservation() {
+        mutableLiveData.value = ParkingStayData(CheckState.SAVE_RESERVATION)
```

Comment:
this should call the model that is the one that have to save the reservation info in the db